golang/cosmos/app/upgrade.go

```diff
@@ -191,6 +191,10 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
                 vm.CoreProposalStepForModules(
                     "@agoric/builders/scripts/vats/revive-kread.js",
                 ),
+				vm.CoreProposalStepForModules(
+					// Upgrade to new liveslots and support vows.
```

Comment:
Im not sure this comment is still relevant

Reply:
doh!   of course not.